client/src/components/AuditAdmin/Progress/Progress.test.tsx

```diff
@@ -325,14 +326,29 @@ describe('Progress screen', () => {
     })
   })
 
-  it('shows round status for batch comparison', async () => {
+  it('shows round and discrepancy status for batch comparison', async () => {
```

Comment:
This test was already checking discrepancy counts and downloading the discrepancy report, so it seemed like a good place to also test the discrepancy modal